README.md

```diff
@@ -163,6 +163,11 @@ docker run -v $(pwd):/app ganler/evalplus:latest --dataset [humaneval|mbpp] --sa
 ```bash
 evalplus.evaluate --dataset [humaneval|mbpp] --samples samples.jsonl
 ```
+when you already download the dataset [MbppPlus.jsonl.gz|HumanEvalPlus.jsonl.gz]:
+```bash
+HUMANEVAL_OVERRIDE_PATH="the local path of HumanEvalPlus.jsonl.gz" evalplus.evaluate --dataset humaneval --samples samples.jsonl
+```
+
```

Comment:
```suggestion
To use a user-defined dataset locally, you can set `HUMANEVAL_OVERRIDE_PATH` or `MBPP_OVERRIDE_PATH`:

```bash
HUMANEVAL_OVERRIDE_PATH="/path/to/HumanEvalPlus.jsonl.gz" evalplus.evaluate --dataset humaneval --samples samples.jsonl
```